cpanfile

```diff
@@ -20,6 +20,7 @@ requires 'Email::Sender::Transport::SMTP';
 requires 'Email::Stuffer';
 requires 'Feature::Compat::Try';
 requires 'File::Find::Rule';
+requires 'File::Grep';
```

Comment:
If this dep is used in `utils/devel/` then shouldn't it be in `on devel =>` as a dependency? That might save the exclusion in `xt/01.2-deps.t` as well.

Reply:
Indeed. Fixed.